cubedash/_pages.py

```diff
@@ -244,6 +244,24 @@ def about_page():
     return utils.render("about.html")
 
 
+@app.route("/dashboard")
+def dashboard_page():
+    dc = datacube.Datacube()
+    dashboard = []
+    for p, _ in _model.get_products_with_summaries():
+        dataset_list = dc.find_datasets(product=p.name, limit=1)
+        uri = dataset_list[0].uris[0]
```

Comment:
Can be empty, which will throw exception

Reply:
thats been changed.